acoustic_msgs/README.md

```diff
@@ -176,17 +169,20 @@ The image details details how image data is stored.
 
 
 ### Geometry Conventions
-![rx_tx_detail](https://user-images.githubusercontent.com/23006525/160200735-2698b659-27f9-4309-ac3f-1b891858ddb3.png)
+![rx_tx_detail](https://user-images.githubusercontent.com/23006525/167165995-ce578962-be79-4219-89a0-920953ca6cb2.png)
 
-* angles/beam directions in the sptaial domain shall:
-  * be reported as unit vectors according to the in the corridate frame convention specified above.   
+
+* angles/beam directions in the spatial domain shall:
+  * be reported as unit vectors according to the coordinate frame convention specified above.   
   * be reported at time reported in the header timestamp.  (at or before transmit time)  
 * angles/beam directions in the temporal domain shall be reported as transmit (Tx) and receive (Rx) angles according to the diagram above
-  * rx_angle is defined as the angle from the y-z plane at receive time. Positive rotation around the z axis.  This angle defines a cone of possible returns directions for a given beam.
-  * tx_anlge is defined as the angle from the x-y plane at transmit time. Positive rotation around the y axis.  This angle defines a cone that represents the area insonified by the tx pulse.  
+  * rx_angle is defined as the elevation from the X-Z plane at receive time. This elevation is positive toward the Y axis.  This elevation defines a cone of possible return directions for a given return beam.
+  * tx_anlge is defined as the elevation from the Y-Z plane at transmit time. This elevation is positive toward the X axis.  This elevation defines a cone that represents the area insonified by the tx pulse.
+
+
 
 ### Style Conventions
-These messages were designed to comply withe the conventions set forth by the [ROS/Patterens/Conventions guide](http://wiki.ros.org/ROS/Patterns/Conventions#Messages).   Beyond that, we have set the following conventions:
+These messages were designed to comply with the conventions set forth by the [ROS/Patterns/Conventions guide](http://wiki.ros.org/ROS/Patterns/Conventions#Messages).   Beyond that, we have set the following conventions:
 * All vector quantities should have plural names
-* data strutures should be represented as a "structures of arrays" rather than "arrays of structures".
-* vectors quantities may be of length zero.  This shall be interpreted as unreported.  (see internal MSG doccumentaiton for more detials)
+* data structures should be represented as a "structures of arrays" rather than "arrays of structures".
+* vector quantities may be of length zero.  This shall be interpreted as unreported.  (see internal MSG documentation for more details)
```

Comment:
As mentioned in an email, I think this is still a bit up for discussion -- I'd prefer to force the publisher to specify sane default values, rather than forcing all consumers of the data to implement that logic.

Reply:
I see advantages to either use case.   I believe this is why we chose the structure of arrays over array of structures though.  That way unreported values could be of length zero to save space.   I agree it adds more work for the processing pipeline though and could potentially cause unexpected bugs.

Maybe this is worth a phone discussion.   

In  the mean time, I have marked this as "provisional" for now though so we can complete the pull request.